seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -129,7 +129,7 @@
         "<blockquote>\"use strict\"<br>const FAV_PET = \"Cats\";<br>FAV_PET = \"Dogs\"; // returns error</blockquote>",
         "As you can see, trying to reassign a variable declared with <code>const</code> will throw an error. You should always name variables you don't want to reassign using the <code>const</code> keyword. This helps when you accidentally attempt to reassign a variable that is meant to stay constant. A common practice is to name your constants in all upper-cases and with an underscore to separate words (e.g. <code>EXAMPLE_VARIABLE</code>).",
         "<hr>",
-        "Change the code so that all variables are declared using <code>let</code> or <code>const</code>. Use <code>let</code> when you want the variable to change, and <code>const</code> when you want the variable to remain constant. Also, rename variables declared with <code>const</code> to conform to common practices, meaning constants should be in all caps"
+        "Change the code so that all variables are declared using <code>let</code> or <code>const</code>. Use <code>let</code> when you want the variable to change, and <code>const</code> when you want the variable to remain constant. Also, rename variables declared with <code>const</code> to conform to common practices, meaning constants should be in all caps. The variable <code> sentence </code> in the <code> console.log </code> statement, needs to be changed to conform with common <code> const </code> practices."
       ],
       "tests": [
         {
```

Comment:
I can't select the exact line for some reason, but I think we should simplify some of the other text and test strings a little bit:

`"<code>SENTENCE</code> should be a constant variable declared with <code>const</code>."`

`"<code>i</code> should be declared with <code>let</code>."`